_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: newsletter
+title: "Neovim News #11 - The Christmas Issue"
+category: newsletter
+permalink: /news/2021/07/09
+---
+
+# Neovim News #11 - The Christmas Issue
+
+> The _real_ 0.5 was the friends we made along the way
+
+
+The long-awaited release of Neovim v0.5.0 finally happened on July 2, 2021. Due to a variety of reasons, both technical and personal (such as a certain world-wide pandemic that everyone is heartily sick of by now), this took longer than everybody had hoped for. But it's a big one! In fact, with over 4000 commits, it is so big that it broke some of the tooling for the release. So these notes will not be able to touch on each of the many changes that were made over the course of development and only focus on the most user-visible improvements, of which the biggest are
```

Comment:
let's not even mention the pandemic, it's cliche. also don't need to make excuses about the delay, though we can acknowledge it briefly.

```suggestion
The long-awaited release of Neovim v0.5.0 finally happened on July 2, 2021. It was delayed too long, but it's worth the wait. With over 4000 commits, it is so big that it broke some of the release tooling. So these notes will not be able to touch on each of the many changes that were made over the course of development and only focus on the most user-visible improvements, of which the biggest are
```

Reply:
We discussed this, and it is a fact that the pandemic had a significant effect on the work (maybe not for everyone, but certainly for some). But it should be taken as given, and I can certainly tone it down.

(I wouldn't write "too long" -- it took as long as it needed to take, given the circumstances.)

Reply:
sure, this also works: "It was delayed, but it's worth the wait."